### ISSUE TITLE
z-index too high

### ISSUE THREAD

**Issue by born2excite** [mozilla/fx-private-relay-add-on#225]
The Relay plugin causes the email address input field sits above all other page elements.

Specifically, when a menu opens on hover, the the email input field will display above the open menu, obscuring the visitors view of the menu. Please see class and z-index details below and attached screenshot.

![Firefox relay z-index problem](https://user-images.githubusercontent.com/10362816/144531488-335749dd-2f20-4dbb-a61b-4db4686ac0f8.png)

```
.fx-relay-email-input-wrapper {
  z-index: 99999999999999999;
}
```

Please consider removing or lowering the z-index or some other fix.

Thank you
Julian

**Comment by einsiol** [comment]
Yes, I'm seeing the same issue, this can be a bit disrupting when navigating on websites where the input will start overlaying the content while scrolling.

**Comment by lolrepeatlol** [comment]
Duplicate of #22
